test/test_unit_kernel.cpp

```diff
@@ -9,17 +9,49 @@
 #include "gtest/gtest.h"
 
 #include <string>
+#include <memory>
 
 #include "nlohmann/json.hpp"
 
+#include "test_interpreter.hpp"
+
 #include "xeus/xkernel.hpp"
 #include "xeus/xmiddleware.hpp"
 #include "xeus/xsystem.hpp"
 
+#include "xeus/xhelper.hpp"
+
 namespace nl = nlohmann;
 
 namespace xeus
 {
+    // TEST(kernel, print_starting_message)
+    // {
+    //     std::string file_name = "connection.json";
+    //     xconfiguration config = load_configuration(file_name);
+
+    //     using interpreter_ptr = std::unique_ptr<test_kernel::test_interpreter>;
+    //     interpreter_ptr interpreter = interpreter_ptr(new test_kernel::test_interpreter());
+    //     xkernel kernel(config, get_user_name(), std::move(interpreter));
+
+    //     print_starting_message(config);
+    // }
+
+    TEST(kernel, extract_filename)
+    {
+        char** argv;
+        argv[0] = "-f connection.json";
+        std::string file_name = extract_filename(1, argv);
+        EXPECT_EQ(file_name, "connection.json");
+    }
```

Comment:
This also fails and I don't understand why because I can't really debug the test outputs?
I'm using `ctest -VV --output-on-failure` but can't see anything. Is there a way of seeing test outputs?